### DISCUSSION TITLE
Unable to hide leaderboard in multiplayer

### DISCUSSION THREAD

**Discussion by timiimit** [ppy/osu#28492] [Q&A] [answered]
In solo pressing Tab allows player to hide leaderboards, the same is not possible in multiplayer. Is this intentional because some other feature is expected in the future or is this a bug?

Here's a video showing unexpected functionality.

https://github.com/ppy/osu/assets/32331609/dc387cb1-c8e4-41f2-8001-5504ed70d45e

**Comment by BlockHead007** [comment, marked as answer]
For now, this is intentional
Latest discussion was at #27615
